deps.edn

```diff
@@ -181,11 +181,11 @@
   ;; Requires nashorn in a version of ClojureScript
   ;; WARNING: stack trace - does not run TO DEPRECATE
   :repl/rebel-figwheel
-  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.9.946"} ;; version contains cljs.repl.nashorn
+  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.11.54"} ;; version contains cljs.repl.nashorn
```

Comment:
This alias is set to a version of Clojurescript that contains cljs.repl.nashorn, which is needed for this alias to work correctly.  Updating the version breaks the alias (and the alias is marked as deprecated, so should not be changed)